src/rules/elseif-without-else.ts

```diff
@@ -24,6 +24,12 @@ import { Rule } from '../utils/types';
 const rule: Rule.RuleModule = {
   meta: {
     type: 'suggestion',
+    docs: {
+      description: '"if ... else if" constructs should end with "else" clauses',
+      category: 'Best Practices',
+      recommended: false,
```

Comment:
should it be `off` to be consistent with `index.ts`? 
Can we rely on this value from `docs` when creating profile in `index.ts`? So that it's not duplicated

Reply:
> should it be `off` to be consistent with `index.ts`?

No, it shouldn't unfortunately. [Here](https://github.com/typescript-eslint/typescript-eslint/blob/f2c078cb1726349e5d1bb252e2653f4eb6310b68/packages/experimental-utils/src/ts-eslint/Rule.ts#L26) is the type of the `recommended` property.

> Can we rely on this value from `docs` when creating profile in `index.ts`? So that it's not duplicated

Yes, we can. I will do that.